english/coffee/nude-run/cold-water/cold-water.md

```diff
@@ -4,3 +4,25 @@ head.
 
 "I suppose," the badger whispers at last, "that even in the light
 of day, the Fundamental Truth can elude me."
+
+This badger is really pissing you off. Why is he saying such 
+stupid enlightend crap like this? You are furious that he could
+be so pretentious. What a piece of work.
+
+The badger notices your frustration, and starts to assume a 
+defensive pose. Honey badger's kick ass, you remember.
+
+You suddenly notice the .38 magnum lying on the ground in front of you. 
+It looks like there is one bullet left. locked and loaded. 
+
+Unfortunately the badger noticed the gun as well. The gun is closer 
+to you, but not by much. You might be able to grab it, But you are not
+sure how quick the badgers reflexes are.
+
+What do you do?
+
+["Strangle the badger with your hands"](../nude-run/cold-water/strangle/strangle.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.